Poisson_surface_reconstruction_3/test/Poisson_surface_reconstruction_3/poisson_reconstruction_test.cpp

```diff
@@ -8,6 +8,10 @@
 // poisson_reconstruction_test mesh1.off point_set2.xyz...
 
 // CGAL
+
+/// @TODO: change the usage from Surface_mesher to Mesh_3
+#include <CGAL/Installation/internal/disable_deprecation_warnings_and_errors.h>
+
 #include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
```

Comment:
Maybe we should keep this test, to test the backward compatibility.

But then it should be duplicated with another one using Mesh_3 instead.